lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -112,7 +117,11 @@ def singleton_method_added(sym) #:nodoc:
       end
 
       def write(text="")
-        puts(text) if verbose
+        @buffer_output ||=  ""
+        if verbose
+          @buffer_output += text + "\n"
```

Comment:
I think we should store the output even if verbose is `false`, verbose is more about the console IMO.